cli/tests/it/cast.rs

```diff
@@ -24,5 +24,5 @@ casttest!(finds_block, |_: TestProject, mut cmd: TestCommand| {
     println!("{}", output);
 
     // Expect successful block query
-    assert!(output.contains("6574364"));
+    // assert!(output.contains("6574364"), "{}", output);
```

Comment:
re-enable

Reply:
this did fail when ETH_RPC_URL was set -.-

Reply:
Weird, let's create an issue and fix later

Reply:
perhaps an issue with the RPC I use locally, let's see